gallery/visualtests/nxToast.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+const { setupBrowser } = require('./testUtils');
+
+describe('NxToast', function() {
+  const openButton = '.nx-btn-bar .nx-btn--secondary';
+  const toastContainer = '.nx-toast-container';
+
+  describe('with global sidebar and header', function() {
+    const {
+      checkFullPageScreenshot,
+      getPage,
+      wait,
+      waitAndGetElements,
+      a11yTest
+    } = setupBrowser('#/NxToastComplexLayoutExample', false);
+
+    async function launchToasts(toastsContainer, buttonSelector) {
```

Comment:
It looks like every invocation of one of these `launchToasts` functions is receiving the same arguments.  They don't really need to be arguments then, `launchToasts` can just refer to `openButton` and `toastContainer` directly.  Additionally it seems like the only difference between the various copies of this function is that they each need to receive different instances of `waitAndGetElements` and `wait` from the closure scope.  Consider making a single copy of the function and passing those functions in as parameters instead.

Reply:
> It looks like every invocation of one of these `launchToasts` functions is receiving the same arguments. They don't really need to be arguments then, `launchToasts` can just refer to `openButton` and `toastContainer` directly. Additionally it seems like the only difference between the various copies of this function is that they each need to receive different instances of `waitAndGetElements` and `wait` from the closure scope. Consider making a single copy of the function and passing those functions in as parameters instead.

So I replaced the arguments being passed in to `wait` and `waitAndGetElements`, and moved it out of each describe here: [98c32a2](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/98c32a25a696ae887bfb4bc9db299df9da53f028)